types/africastalking/africastalking-tests.ts

```diff
@@ -0,0 +1,29 @@
+import AfricasTalking from "africastalking";
```

Comment:
```suggestion
import AfricasTalking = require("africastalking");
```

Reply:
@jakebailey does the above change mean that even when the client uses this library they will have to import it that way?

Reply:
No, just that DT is very strict about getting at the types of libraries; if someone actually was using the esModuleInterop setting, this issue would be smoothed over, more or less.